cosmo_tester/test_suites/bootstrap_based_tests/multi_network_test.py

```diff
@@ -40,6 +40,7 @@
 IPADDR="{1}"
 NETMASK="255.255.255.128"
 DEFROUTE="no"
+MTU=1450
```

Comment:
maybe leave a comment that this should be picked up by DHCP, but is hardcoded here.
Just in case in the future we run this in an env with MTU=1400, and this becomes a problem again